src/core/components/operation.jsx

```diff
@@ -158,7 +160,7 @@ export default class Operation extends React.Component {
                   { summary }
                 </div>
             }
-
+            <span className="opblock-summary-operation-id">{operationId}</span>
```

Comment:
Can we put this in a ternary operator? Something like `{ operationId ? <span className="opblock-summary-operation-id">{operationId}</span> : null }`. Will keep the DOM cleaner when an operationId isn't present.